test/ImportCommand.js

```diff
@@ -171,7 +171,33 @@ describe("ImportCommand", () => {
       importCommand.runPreparations();
 
       importCommand.runCommand(exitWithCode(1, (err) => {
-        const expect = `Target directory already exists "${normalize(targetDir)}"`;
```

Comment:
Which would you prefer?

* Output native paths in the error message
* Coerce to Unix-style even on Windows?

In `master` now, we hard-code UNIX style slashes in `ImportCommand.targetDir`:

```
    this.targetDir = "packages/" + externalRepoBase;
```

… so, in any test checking `targetDir` either explicitly or implicitly via an `Error`, we need to coerce the test's expectation to UNIX style paths.

In my change, I use `path.join` for `targetDir`, so we need to stick with whatever `path.relative` or `path.join` gave us.